css/portal-dashboard/level-viewer.less

```diff
@@ -1,5 +1,9 @@
 @import "../variables";
 
 .levelViewer {
-  height: 240px;
+  height: 205px;
+  width: 100%;
```

Comment:
The 100% width does cause the left class nav component size to get messed up.  I'll see if there's an obvious solution.

Reply:
This works as an alternative so the class nav still behaves the same:
`width: calc(100vw - 240px);`

Reply:
But long term we might need to think about how these components slot together.